### PR TITLE
Add pigz@2.8

### PR DESCRIPTION
**Pull request by metti**
This is mostly a copy-upgrade from 2.7 that brings the module and its dependencies up to date. Yet, I decided to replace the pigz `cc_library` by a `cc_binary` as that is what pigz actually is and that also makes the binary usable via the label `@pigz`. For backwards compatibility, I kept an alias "bin".

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @alexeagle, modules you maintain (pigz) have been updated in this PR. Please review the changes.

**Comment by metti** [comment]
Cc: @alexeagle, the author of 2.7 of this module

**Review by Wyverald — APPROVED** [review]

**Comment by Wyverald** [comment]
I'll wait for @alexeagle to give the final approval.

**Comment by Wyverald** [comment]
ping @alexeagle

**Review by alexeagle — APPROVED** [review]

**Pull request merged**
